client/src/pages/Home/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC, useEffect } from 'react'
+import { makeStyles, createStyles } from '@material-ui/core'
+import Header from '../../components/Header/Header'
+import Footer from '../../components/Footer/Footer'
+import { useActions } from '../../hooks/useActions'
+import MessageSnackbar from '../../components/MessageSnackbar/MessageSnackbar'
+
+export const useStylesHome = makeStyles(() =>
+    createStyles({
+        wrapper: {
+            display: 'flex',
+            flexDirection: 'column',
+            minHeight: '100vh',
+        },
+        content: {
+            backgroundColor: 'white',
+            flex: '1 0 auto',
+        },
+    })
+)
+
+const Home: FC = () => {
+    const classes = useStylesHome()
+    const { auth } = useActions()
+
+    useEffect(() => {
+        const token = localStorage.getItem('token')
+        if (token) {
+            auth(token)
+        }
+    })
```

Comment:
don't forget to add an empty array here `[]` as the second argument, because you forget to add one it will be able to cause a lot of requests

Reply:
fix